contrib/templates/stats.html.tmpl

```diff
@@ -11,20 +11,25 @@ table, th, td {
 </style></head><body><h1>Hockeypuck OpenPGP Keyserver Statistics</h1>
 Taken at {{ .Now }}
 <h2>Settings</h2>
-<table>
-<tr><th>Version</th><td>{{ .Version }} </td></tr>
-{{ if .Contact }}<tr><th>Server Contact</th><td>{{ .Contact }} </td></tr>{{ end }}
-<tr><th>HTTP</th><td>{{ .HTTPAddr }} </td></tr>
-<tr><th>Recon</th><td>{{ .ReconAddr }} </td></tr>
+     <table summary="Keyserver Settings">
+     <tr><td>Hostname:</td><td>{{ .Hostname }}</td></tr>
+     <tr><td>Nodename:</td><td>{{ .Nodename }}</td></tr>
+     <tr><td>Version:</td><td>1.1.6+</td></tr>
```

Comment:
This isn't correct. Can we use {{ .Version }}?

Reply:
Sadly no, sks-keyservers.net will consider the server invalid with the actual version. I'm guessing K_F rejects versions > 1.1.6 in his script...